Assets/__Scripts/Localization/Localization.asset

```diff
@@ -696,3 +696,20 @@ MonoBehaviour:
   - you got games on your phone?
   - i like this one
   - 711nedeaC
+  - Made in C# becuz C++ will kill me
+  - The Offical Unoffical Mapper
```

Comment:
Are `Official` and `Unofficial` typos intentional?

Reply:
If you look inside the loading message suggestion channel, I added these messages word for word. I will fix the typos in my next pr to resolve this error and other errors